### PR TITLE
Fix extraArgs typo

### PR DESCRIPTION
**Pull request by 3846masa**
#33 has typo, so it doesn't work.

This PR,
- fix typo
- add default args
- update example `webpack.config.js`

### CODE REVIEW

File: plugin.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ let ranInitialCompilation = false;
 class WasmPackPlugin {
   constructor(options) {
     this.crateDirectory = options.crateDirectory;
-    this.extraArgs = (options.extraArgs || '').trim().split(' ').filter(x=> x);
+    this.extraArgs = (options.extraArgs || '--target browser --mode normal').trim().split(' ').filter(x=> x);
```
Comment (robertohuertasm) [inline plugin.js]: This is not needed at all. These values are the default `wasm-pack` values.

File: README.md [hunk 2]
```diff
@@ -25,8 +25,8 @@ module.exports = {
     new WasmPackPlugin({
       crateDirectory: path.resolve(__dirname, "crate"),
       // check https://rustwasm.github.io/wasm-pack/book/commands/build.html for extraArgs
-      // defautls --typescript --target browser --mode normal
-      extraArgs: "--no-typescript --mode no-install",
+      // defaults `--target browser --mode normal`
```
Comment (robertohuertasm) [inline README.md]: Please, add `--typescript` again as it is a default value for the `wasm-pack build` command.

File: plugin.js (hunk 1 above)
Comment (3846masa) [reply plugin.js]: Thanks, I'll remove.

File: README.md (hunk 2 above)
Comment (3846masa) [reply README.md]: I think `wasm-pack` doesn't support `--typescript` arg. It cause confusion.

https://github.com/rustwasm/wasm-pack/blob/600d02b292083fdd1d7a7ed1bc3e429048feed2c/src/command/build.rs#L78-L81

Comment (robertohuertasm) [reply README.md]: You're totally right! My bad. I saw [this line](https://github.com/rustwasm/wasm-pack/blob/600d02b292083fdd1d7a7ed1bc3e429048feed2c/src/bindgen.rs#L145) and it led me to confussion. 👍

**Pull request merged**